nms/app/views/equipment/GatewayDetailSubscribers.tsx

```diff
@@ -51,15 +44,19 @@ export default function GatewayDetailSubscribers(props: GatewayDetailType) {
     refresh: props.refresh,
   });
   const subscriberCtx = useContext(SubscriberContext);
-  const gwSubscriberMap =
-    // $FlowIgnore
-    subscriberCtx.gwSubscriberMap[props.gwInfo?.device?.hardware_id] || [];
+  const hardware_id = props.gwInfo?.device?.hardware_id;
+  const gwSubscriberMap = hardware_id
+    ? subscriberCtx.gwSubscriberMap[hardware_id]
+    : [];
 
   console.log('gwSubscriberMap: ', gwSubscriberMap);
```

Comment:
The first linter warning I see in the code here.
Should we address this at the same time, or is this out of scope?

Reply:
I would say out of scope. Once flow is gone we can turn warnings into errors and fix them as a separate story.